fdbserver/Knobs.cpp

```diff
@@ -201,6 +201,7 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs, bool isSimula
 	init( DD_OVERLAP_PENALTY,                                  10000 );
 	init( DD_VALIDATE_LOCALITY,                                 true ); if( randomize && BUGGIFY ) DD_VALIDATE_LOCALITY = false;
 	init( DD_CHECK_INVALID_LOCALITY_DELAY,                       60  ); if( randomize && BUGGIFY ) DD_CHECK_INVALID_LOCALITY_DELAY = 1 + deterministicRandom()->random01() * 600;
+	init( DD_ENABLE_VERBOSE_TRACING,                           false ); if( randomize && BUGGIFY ) DD_ENABLE_VERBOSE_TRACING = true;
```

Comment:
I wonder whether this should be true by default? Or if not I would at least make it true in simulation (as there the number of trace lines is less of a problem).

Reply:
It is true for BUGGIFY. So it should be tested in simulation.

The Trace severity level is always SevInfo instead of SevDebug for frequent events in the current code, which could produce more such events than the current FDB release.

I'm not 100% sure about how frequent this event will be output, although I guess it should not produce a lot. So set it as false by default. If we confirmed in prod that it is not many, we can toggle it later.

Reply:
Ah - I missed that this is `DevDebug` currently, in that case I agree that it should probably be set to false for now (we could potentially set the default to true in master later if all goes well).